OpenRA.Game/Graphics/Palette.cs

```diff
@@ -97,18 +97,18 @@ public void CopyToArray(Array destination, int destinationOffset)
 			Buffer.BlockCopy(colors, 0, destination, destinationOffset * 4, Palette.Size * 4);
 		}
 
-		public ImmutablePalette(string filename, int[] remap)
+		public ImmutablePalette(string filename, int[] remap, int background = 0, uint color = 0)
```

Comment:
It seems rather silly to add this extra plumbing in order to support a palette loaded from a custom file.  Just build the custom palette in code instead - like we do for our other special palette types.